src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -393,6 +393,19 @@ <h4 class="modal-title" id="onlineCourseConfigurationTitle" jhiTranslate="artemi
                                 </div>
                             </ng-container>
                         </div>
+                        <div class="form-group">
+                            <label class="form-control-label" jhiTranslate="artemisApp.course.onlineCourseConfiguration.requireExistingUser" for="field_requireExistingUser"
+                                >Require Existing User</label
+                            >
+                            <input
+                                type="checkbox"
+                                class="form-check-input"
+                                name="requireExistingUser"
+                                id="field_requireExistingUser"
+                                [checked]="course.onlineCourseConfiguration?.requireExistingUser"
+                                formControlName="requireExistingUser"
+                            />
+                        </div>
```

Comment:
All other checkboxes in this edit view have the label on the right of the checkbox.
Ad you probably should add a jhi-help icon with a tool tip explaining this setting.